main.py

```diff
@@ -1,7 +1,49 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create([
+        Genre(name="Western"),
+        Genre(name="Action"),
+        Genre(name="Dramma"),
+    ])
+    Actor.objects.bulk_create([
+        Actor(first_name="George", last_name="Klooney"),
+        Actor(first_name="Kianu", last_name="Reaves"),
+        Actor(first_name="Scarlett", last_name="Keegan"),
+        Actor(first_name="Will", last_name="Smith"),
+        Actor(first_name="Jaden", last_name="Smith"),
+        Actor(first_name="Scarlett", last_name="Johansson"),
+    ])
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+
+    ).update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(
+        first_name="Keanu",
+        last_name="Reeves"
+    )
+
+    Genre.objects.filter(
```

Comment:
Try to break down main function is smaller specialized functions to structure your code better. E.g separate function for creating, updating, deleting (no need to parametrize arguments, it will be enough to keep expressions as they are right now, just group them under different functions)